src/extras/helpers/ArrowHelper.js

```diff
@@ -94,6 +92,9 @@ THREE.ArrowHelper.prototype.setLength = function ( length, headLength, headWidth
 	if ( headLength < length ){
 		this.line.scale.set( 1, length - headLength, 1 );
```

Comment:
Would this work -- instead of `if-then-else` and toggling visibility?

     this.line.scale.set( 1, Math.max( 0, length - headLength ), 1 );